sources/api/servicedog/src/main.rs

```diff
@@ -24,16 +24,11 @@ use std::env;
 use std::ffi::OsStr;
 use std::process::{self, Command};
 use std::str::FromStr;
+use constants;
```

Comment:
$1.00 if we delete `servicedog` since we haven't used it for anything in over a year 😀

Reply:
You will owe me $2 dollars :money_mouth_face: